Wikipedia/Code/AccountViewController.swift

```diff
@@ -160,6 +164,29 @@ class AccountViewController: SubSettingsViewController {
             break
         }
     }
+
+    @objc func showWatchlistOnboarding() {
```

Comment:
@mazevedofs Just a heads up, I notice this doesn't theme. I actually worked on communicating theming stuff from the client app back in https://github.com/wikimedia/wikipedia-ios/pull/4521, so I think merging that PR will fix it. Also heads up to @staykids who recently asked about this - looks like that work is done in that PR.